src/domain_factory/accountcollection/account_group_factory.py

```diff
@@ -0,0 +1,30 @@
+from domain_factory.accountcollection.account_collection_factory import \
+    AccountCollectionFactory
+
+
+class AccountGroupFactory(AccountCollectionFactory):
+    """ Class to generate Account Groups.  Generate method will call the
+        AccountCollectionFactory generate method in order to populate fields
+        shared by all Account Collections.  The generate method in this class
```

Comment:
can we have the lines of these comments start in line with the first line of the comment? You should be able to highlight the required section and use 'shift+tab'